frequency-chain/frequency-xyz
# Use dark footer

# Description

Update all routes to use newdark (blue #14313F background) footer as per design

- Closes #50 

## Screenshot(s)

<img width="920" alt="image" src="https://github.com/user-attachments/assets/e965f220-ece6-44a5-8749-55dfcbbe0043">

<img width="920" alt="image" src="https://github.com/user-attachments/assets/a8796455-6b48-43f1-8dc6-c7c8f674957f">

## Type of change

Please delete options that are not relevant.

      re-approval.)

# How to Test?

1. Pull and run locally
2. Go to "About" and "Dev Docs" links
3. See that footer is dark blue and lines up with content

# Checklist:

- [ x I have commented my code & PR, particularly in hard-to-understand areas
